docs/topics/forms/index.txt

```diff
@@ -372,7 +372,9 @@ More about Django :class:`Form` classes
 
 All form classes are created as subclasses of :class:`django.forms.Form`,
```

Comment:
All form classes (except ModelForm) are created as subclasses of :class:`django.forms.Form`.

Reply:
I revised the wording, let me know what you think.